docs/release/3.5/release-3-5-0.md

```diff
@@ -0,0 +1,145 @@
+OSG Software Release 3.5.0
+===========================
+
+**Release Date:** 2019-08-30    
+**Supported OS Versions:** EL7
+
+Summary of changes
+------------------
+
+This initial release of the OSG 3.5 release series is based on the packages available in
+[OSG 3.4.33](/release/3.4/release-3-4-33#) with some [additions](#package-updates) and [subtractions](#package-removals).
+Other notable changes in this release series include dropping support for Enterprise Linux 6 and
+[CREAM CEs](https://opensciencegrid.org/technology/policy/cream-support/).
+
+To update to the OSG 3.5 release series, please consult the page on
+[updating between release series](/release/release_series#updating-to-osg-35).
+
+!!! question "Where are GlideinWMS and HTCondor-CE?"
+    HTCondor-CE (including `osg-ce` metapackages) and GlideinWMS are both absent in OSG 3.5.0:
+    we expect major version upgrades that may require manual intervention for both of these packages so we are holding
+    their initial releases in this series until they are ready.
+
+!!! warning "OSG 3.4 end-of-life"
+    As a result of this initial OSG 3.5 release, the end-of-life dates have been set for OSG 3.4 per our
+    [policy](https://opensciencegrid.org/technology/policy/release-series/):
+    regular support will end in **February 2020** and critical bug/security support will end in **November 2020**.
+
+### Package updates ###
+
+In addition to the packages that were carried over from [OSG 3.4.33](/release/3.4/release-3-4-33#),
+this release contains the following package updates:
+
+-   HTCondor 8.8.4: See the [manual upgrade instructions](/release/release_series#updating-to-htcondor-8.8.x) before
+    updating to this version.
+-   CVMFS 2.6.2: A [bug fix release](https://cvmfs.readthedocs.io/en/2.6/cpt-releasenotes.html).
+    Note the upgrade recommendations from the developers:
+
+    > As with previous releases, upgrading clients should be seamless just by installing the new package from the
+    > repository.
+    > As usual, we recommend to update only a few worker nodes first and gradually ramp up once the new version proves
+    > to work correctly.
+    > Please take special care when upgrading a cvmfs client in NFS mode.
+
+    > For Stratum 1 servers, there should be no running snapshots during the upgrade.
+    > For publisher and gateway nodes, all transactions must be closed and no active leases must be present before
+    > upgrading.
+
+    See the known issue with this version [below](#known-issues).
+
+-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
+-   OSG Configure 3.0.0: A [major version release](https://github.com/opensciencegrid/osg-configure/releases/tag/v3.0.0).
```

Comment:
@matyasselmeci are there any interesting things we can say about osg-configure here for our admins?

Reply:
The bosco features added in 2.4.0/2.4.1 are also in 3.0.0 (though probably only hosted CE folks will care). Otherwise, deprecated features were dropped, and some manual config changes may or may not be necessary.

Reply:
Thanks!